### DISCUSSION TITLE
The problem that the reconstructed variant of high temperature deformed alloy does not conform to Burgers relation

### DISCUSSION THREAD

**Discussion by Cici-555** [ORTools4MTEX/ORTools#11] [Q&A] [answered]
![1](https://user-images.githubusercontent.com/103178862/164627215-e248a4c1-6378-403a-96e5-b2fc339cdee0.jpg)
Dear Frank Niessen,
Good evening, I have a question that I would like to ask you. When I reconstructed the titanium alloy deformed at high temperature, I found that the parent phase and the child phase did not conform to the Burgers relationship. I would like to ask whether the variant at this time is considered as deviating from or close to Burgers relationship.

**Comment by AzdiarGazder** [comment, marked as answer]
Hi Cici-555,

Given that I do not have your experimental data to compare with, I'll try my best to answer your question.

To begin the parent grain reconstruction process, we need to "guess" what the operative OR was. A good choice here is a rational OR.

A rational OR is any one of the well known ORs (K-S, N-W, Burgers etc) that comprise geometrically perfect relationships.

This initial guess is then improved on by iterative refinement based on the map data you have. And this refined OR is considered to be the "operative OR". This is used to perform the reconstruction.

Given the differences between them, there will be a deviation between the rational and operative ORs. The question you need to ask yourself is "By how much are these 2 ORs deviated?" A few degrees of disorientation is completely acceptable.

Please have a look at Example 1 and its video - an explanation of the OR disorientation plot is provided there.

Hope this helps.

Warm regards,
Azdi
